packages/priority-queue/src/main/PriorityQueue.ts

```diff
@@ -29,13 +29,16 @@ export class PriorityQueue {
       }
       return b.priority - a.priority;
     },
-    maxRetries: Infinity,
+    maxRetries: 0,
```

Comment:
Changes the default `maxRetry` to 0 since this is not a `InfinityQueue` or `RetryingQueue` and Infinity can cause unexpected behaviour as default value.